core/util/google.py

```diff
@@ -17,16 +17,18 @@
 
 import re
 import os
-
+from core.util import proxy
 class main:
 	# framework = None
-	def __init__(self, q, limit=1, count=10, google_api=None, google_cx=None):
+	countip=0
```

Comment:
not here

Reply:
Sir I didn't get what you meant here. 😅

Should I remove `countip=0`? If yes, how do I replace it? 😅
It'll be really helpful if you could tell me.

Reply:
Define it in init

Reply:
Done.